content/en/xlt/load-testing/Manual/570-scorecard.md

```diff
@@ -9,7 +9,210 @@ description: >
+    "name": "Homepage C", // the name of the rule - optional
+    "comment": "Lorem ipsum ...", // some comment; won't be evaluated - optional
+    "enabled": true, // whether the rule should be evaluated at all (helps to add and remove something easily) - optional, default = true
+    "description" : "Homepage C rating", // the descriptive text to display in the report
+    "failsTest": true, // when true, can make the load test be marked as failed independent of a rating fail - optional, default = false
+    "negateResult": false, // when this is 'true' the rule will fail when all checks do pass and the rule will succeed when at least one check fails - optional, default = false
+    "testFailTrigger": "NOTPASSED", // rule evaluation status used as trigger when to mark test as failed (has no effect unless "failsTest" is "true") - optional, default = NOTPASSED
+    // the checks to run - optional
+    "checks" : [
+        {
+            "selector": "//requests/request[name[text() = 'Homepage']]/percentiles/p95", //XPath
+            "condition": "<= 1000", //XPath
+            "displayValue" : true, // whether the item matching the selector should be displayed in the report - optional, default = true
+            "enabled": true // helps to disable a check easily, if desired - optional, default = true
+        },
+        {
+            "selectorId": "homepageRuntimeP99", //predefined selector
+            "condition": "<= 3000"
+        }
+    ],
+    // message to display in report when rule does pass/fail - optional, no default value, allowed properties: "success" and "fail"
+    "messages": { 
+        "success" : "C", // use this message in the report when passed - optional, no default value
+        "fail" : "", // otherwise print this text - optional, no default value
+    },
+    "points" : 5 // the number of points to achieve - optional, default = 0, no upper limit
+}
+```
+
+### Checks
+
+Each check is a selector against the XML tree of the testreport.xml using XPath. Rule checks can only have a `selector` or a `selectorId`, not both. Everything selected must be either true or false, a number or a text, or a list of text nodes that later can match a regular expression.
+
+Here are a few examples for checks:
+
+```json
+"checks" : [
+    {
+        // Fetch a number
+        "selector": "//requests/request[name[text() = 'Homepage']]/percentiles/p95",
+        "condition": "<= 1000"
+    },
+    {
+        // Fetch Nodes by Regex
+        "selector": "count(//transactions/transaction/name[matches(text(), 'T.*Cart')])",
+        "condition": "> 0"
+    },
+    {
+        // Compare Against a Number
+        "selector": "count(//agents//totalCpuUsage/mean[number() > 90])",
+        "condition": "= 0"
+    }
+```
+
+### Selectors
+
+It is often helpful to define a set of reusable expressions to query the test result document. These are selectors, which are defined as follows:
+
+```json
```

Comment:
JSON does not support comments. Either we have to use js as language for highlighting or remove the comments.

```suggestion
```js
```